README.md

```diff
@@ -1,14 +1,160 @@
 # Lightbox
 
-[![CI Status](http://img.shields.io/travis/hyperoslo/Lightbox.svg?style=flat)](https://travis-ci.org/hyperoslo/Lightbox)
 [![Version](https://img.shields.io/cocoapods/v/Lightbox.svg?style=flat)](http://cocoadocs.org/docsets/Lightbox)
+[![CI Status](http://img.shields.io/travis/hyperoslo/Lightbox.svg?style=flat)](https://travis-ci.org/hyperoslo/Lightbox)
+[![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![License](https://img.shields.io/cocoapods/l/Lightbox.svg?style=flat)](http://cocoadocs.org/docsets/Lightbox)
 [![Platform](https://img.shields.io/cocoapods/p/Lightbox.svg?style=flat)](http://cocoadocs.org/docsets/Lightbox)
+![Swift](https://img.shields.io/badge/%20in-swift%203.0-orange.svg)
+
+**Lightbox** is a convenient and easy to use image viewer for your iOS app,
+packed with all the features you expect:
+
+- [x] Paginated image slideshow.
+- [x] Video support.
+- [x] Double-tap to zoom.
+- [x] Image caption.
+- [x] Dynamic background.
```

Comment:
Should we describe what `Dynamic background` is? Or is that clear enough in itself?